docs/Developers/Development_Setup.rst

```diff
@@ -58,11 +58,20 @@ in your current directory, as coala only works for Python >= 3.4.4
     $ virtualenv coala-venv -p $(which python3)
 
 - Run coala-venv :
+    + If you are a Linux user:
+
     ::
 
         $ source coala-venv/bin/activate
         (coala-venv)$ deactivate # to exit the environment
 
+    + And if you are a Windows user:
+
+    ::
+
+        $ coala-venv\Scripts\activate.bat
```

Comment:
'$' is used to signify a bash prompt. Maybe use '>' for Windows?

Reply:
Resolved.